Unity-Technologies/ScriptableRenderPipeline
# 2d/preview window fix

### Purpose of this PR
Fixes a problem with sprites rendering with lighting in preview windows and in the tilemap window. In the tilemap window tiles would appear black because they were unaffected by scene lighting.

---
### Release Notes
- Fixed problem where tilemap tiles would appear black when using the 2D renderer
- Fixed problem where sprites in the preview window would be lit by 2d scene lighting.

---
### Testing status
Almost all green. There is a failure on Test Editor - Windows. However, in the logs, all the test seemed to have passed.

**Manual Tests**: What did you do?
- [ Y] Opened test project + Run graphic tests locally
- [ Y] Built a player
- [NA ] Checked new UI names with UX convention
- [ NA] Tested UI multi-edition + Undo/Redo
- [ Y] C# and shader warnings (supress shader cache to see them)
- [ NA] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)
No new tests were added.

Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=master&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

Any test projects to go with this to help reviewers?
No.

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low. Relatively straightforward bug fixes with few files changed.

**Halo Effect**: None, Low, Medium, High?
Low. Code change should only be running when 2D Renderer is selected.
---
### Comments to reviewers
Notes for the reviewers you have assigned.
